app/assets/javascripts/angular/directives/query/query_details_directive.js.es6

```diff
@@ -3,14 +3,15 @@
 
   class QueryDetailsController {
     constructor(QueryHandler, QueryTab, TagResource, defaultDateFormat, DefaultAceConfigurator, OpenReplService,
-      RoleModel) {
+      RoleModel, $window) {
```

Comment:
for navigating, prefer the `$location` service over `$window`

https://github.com/lumoslabs/aleph/blob/master/app/assets/javascripts/angular/services/query/query_handler.js.es6#L29

also we might want to just put this in query_handler service